algebird-test/src/test/scala/com/twitter/algebird/MinHasherTest.scala

```diff
@@ -1,20 +1,21 @@
 package com.twitter.algebird
 
+import com.twitter.algebird.BaseProperties._
 import org.scalacheck.{ Arbitrary, Gen }
 import org.scalatest.{ Matchers, _ }
+import scala.math.Equiv
 
 class MinHasherTest extends CheckProperties {
-  import com.twitter.algebird.BaseProperties._
-
   implicit val mhMonoid = new MinHasher32(0.5, 512)
   implicit val mhGen = Arbitrary {
     for (
       v <- Gen.choose(0, 10000)
     ) yield (mhMonoid.init(v))
   }
 
-  property("MinHasher is a Monoid") {
-    monoidLawsEq[MinHashSignature]{ (a, b) => a.bytes.toList == b.bytes.toList }
+  property("MinHasher is a commutative monoid") {
+    implicit val equiv: Equiv[MinHashSignature] = Equiv.by(_.bytes.toList)
```

Comment:
this broke binary compatibility when I tried to create a companion object for `MinHashSignature`.